.github/workflows/smoke-test.yml

```diff
@@ -56,6 +56,13 @@ jobs:
     steps:
       - name: Info block
         uses: medyagh/info-block@main
+      - name: Disable spotlight indexing (macos)
+        if: contains(matrix.os, 'macos')
+        run: |
+          # Disable indexing globally
+          sudo mdutil -a -i off
+          # Erase Existing Indexes
+          sudo mdutil -E /
```

Comment:
This is not required to fix the test but a good change, it keeps the load minimal before test.